README.md

```diff
@@ -307,6 +307,22 @@ See the [repository](https://github.com/orhun/git-cliff-action) for other [examp
 
 Also, see the [continuous deployment workflow](./.github/workflows/cd.yml) of this project which sets the release notes for GitHub releases using this action.
 
+There is another option of GitHub Actions: [setup-git-cliff](https://github.com/kenji-miyake/setup-git-cliff).
```

Comment:
Can we mention how this action is different from the `git-cliff-action` with a couple of words? (e.g. it uses binary releases instead of docker etc.)

Reply:
Sure! Thank you for your prompt reply. :smile:

Reply:
@orhun How about this? 70276c89da8766d35a06b1599ab65c8fc7a0e719

Reply:
Perfect 💯